extensions/elasticsearch/wazuh-elastic6-template-alerts.json

```diff
@@ -231,6 +231,186 @@
             }
           }
         },
+        "win": {

```

Comment:
`win` fields should be inside the `data` block in the template. In a lower level of the JSON structure to be consistent with the decoded fields.